uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/suite/WithoutServerTests.java

```diff
@@ -40,7 +39,6 @@
 	DebuggerTest.class,
 	DownloadServerTest.class,
 	FuseProjectTest.class,
-	JMXNavigatorTest.class,
```

Comment:
just to better understand: the test is removed from Smoke and WithoutServer test suite.
What is the reason, it takes too much time?
As soon as there is a requirement we can't have them in these test suites?

Reply:
WithoutServerTests is executed without any requirements. Since JMXNavigatorTest now requires a requirement for a Camel example, I had to remove the test fro the test suite.

In the future we plan to rewrite and remove some test suites according to the integration with Polarion.